support/index.html

```diff
@@ -120,8 +120,8 @@ <h1 class="navbar-brand">
       });
     </script>
     <script>
-      var clientId = 'BWDP9XS89CJq1w6Nzq7iFOHsTh6ChS2b';
-      var domain = 'brucke.auth0.com';
+      var clientId = 'TgqxTDJFG2G3H5TjqvIEEVL5i6IcZGye';
```

Comment:
given this is public, should we keep brucke for playground purposes?

Reply:
Ugh, this was mistakenly committed. Will remove.